apps/backend/src/modules/groups/group.service.ts

```diff
@@ -46,8 +46,17 @@ class GroupService implements Service {
 		return item.toObject();
 	}
 
-	public delete(): ReturnType<Service["delete"]> {
-		return Promise.resolve(true);
+	public async delete(id: number): Promise<boolean> {
+		const group = await this.groupRepository.find(id);
+
+		if (!group) {
+			throw new GroupError({
+				message: ExceptionMessage.GROUP_NOT_FOUND,
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
```

Comment:
Do we really need this check? Maybe lets just return `true` if group was deleted and `false` otherwise?

Reply:
At least, you can avoid `find` request, and call `delete` right away and then check if result is `false` (item was not deleted), that means that there is no item with this id and we can throw error